contracts/interfaces/modules/licensing/ILicensingModule.sol

```diff
@@ -138,4 +138,21 @@ interface ILicensingModule is IModule {
         uint256 licenseTermsId,
         Licensing.LicensingConfig memory licensingConfig
     ) external;
+
+    /// @notice pre-compute the minting license fee for the given IP and license terms.
+    /// the function can be used to calculate the minting license fee before minting license tokens.
+    /// @param licensorIpId The IP ID of the licensor.
+    /// @param licenseTemplate The address of the license template.
+    /// @param licenseTermsId The ID of the license terms.
+    /// @param amount The amount of license tokens to mint.
+    /// @param receiver The address of the receiver.
+    /// @param royaltyContext The context of the royalty.
```

Comment:
Please add return values in the comments here for clarity.

Reply:
Thanks for review! 
Added comments.